schedule/yast/installer_extended.yaml

```diff
@@ -0,0 +1,40 @@
+---
+name: installer_extended
+description: >
+  Test suite performs additional UI checks. As of now following is testes:
+     - License has to be accepted and there is pop-up with a hint;
+     - License translations (except SLE 15 due to missing translations in the
+       beta phase).
+vars:
+  CHECK_PRESELECTED_MODULES: '1'
+  CHECK_RELEASENOTES: '1'
+  EULA_LANGUAGES: korean
+  EXIT_AFTER_START_INSTALL: '1'
+  EXTRABOOTPARAMS: Y2STRICTTEXTDOMAIN=1
+  INSTALLER_EXTENDED_TEST: '1'
+  VALIDATE_CHECKSUM: '1'
```

Comment:
As the vars are defined in test suite, do you really need to define them here as well?

Reply:
As we have parallel ticket to reduce the amount of Test Suite settings, I'm trying to define which variables could be left in yaml schedule or moved to Job Group settings. In cases when it is clear for me, I do that. Just to reduce the amount of work in future. 

So, in this case we'll be able to remove Test Suite settings if the test suite is not used in other Job Groups. But again, it is another task.

Reply:
i thought so. no problem in general. but still i am so if it will save much of time. Can you resolve the conversation for me?